xcell/cls/data.py

```diff
@@ -40,9 +41,149 @@ def __init__(self, data_path='', data={}, override=False,
 
         os.makedirs(self.data['output'], exist_ok=True)
         self._check_yml_in_outdir(override, ignore_existing_yml)
-        self.tr_matrix = None
+        self.tracers = {'wsp': None, 'no_wsp': None}
```

Comment:
give this a different name

Reply:
I was not very imaginative so I couldn't come up with anything better. I wrote a few comments though.